blocks/page_list/templates/itinerary/Itinerary.jsx

```diff
@@ -25,7 +29,7 @@ export default class Itinerary extends React.Component {
 
   constructor(...args) {
     super(...args);
-    this.state = props.itinerary || getItinerary();
+    this.state = this.props.itinerary || getItinerary();
```

Comment:
Yeah - if props isn't being used, the `...args` is an easy call because it's a pretty standard pattern (either construct with special arguments, or pass the arguments to the parent). When you are using props, it's fine to just say `props` on both. I was including it as future-proofing but it's totally fine to just pass `props`.

The original comment wasn't about style but a functional defect, since this.props will be uninitialized and therefore not work.